packages/docs/docs/fhir-datastore/patient-deduplication/patient-deduplication.md

```diff
@@ -0,0 +1,193 @@
+
+We've organized the guide as follows:
+
+- First, we review the high-level architecture of a patient deduplication pipeline
+- Then we talk about how to represent duplicate patients in FHIR
+- Next, we discuss the tradeoffs for building batch vs. incremental pipelines.
+- Finally, we go into some detail on the specific rules that you can use to perform deduplication
+
+## Architecture Overview
+
+### Terminology
+
+Before we get started, let's define some terminology.
+
+In this guide, we will call the output of your deduplication pipeline the **target system**. Each input system that contributes patient data will be called a **source system.**
+
+Most pipelines make a copy of each patient record from the source system into the target system. We will call these copies **source records.** The final, combined patient record will be called the **master record**.
+
+### Pipeline
+
+While deduplication pipelines can take many forms, there three primary operations in every pipeline:
+
+1. **Ingestion:** Copying data from source systems into the target system to create source records.
+2. **Matching:** For each patient record, find all other records that could are potential matches.
+3. **Merging:** Merge all the information into a single record to serve as the source of truth.
+
+There are many different ways to implement this pipeline, but there are a few key decisions that you will need to make when choosing an architecture:
+
+- Whether you are building a batch vs. incremental deduplication pipeline
+- Which matching rules to use
+- How to merge matching patients into a single record
+
+### Decision Factors
+
+We'll discuss each of these in depth. The most important factors to consider when making these decisions are:
+
+- **Prevalence of duplicates:** Making estimate of your duplication rate, both _within_ and _between_ source systems, will help you determine the complexity needed in your deduplication pipeline. Some applications have rare duplication rates (e.g. < 1%) while others have frequent duplicates (10-50%). Still others have very bi-modal duplication patterns (e.g 99% of records have 0 duplicates, 1% of records have 5+ duplicates).
+
+- **Duration of source systems:** Some pipelines are built to ingest patient data from short-lived source systems, with the target system quickly replacing them as the source of truth. Examples include migrating from legacy systems into medplum, ingestion from one-time data sources. In contrast, some source systems are long lived or permanent. This is the case when deduplicating patients from multiple healthcare providers, or when building pipelines from Health Information Exchanges (HIEs).
```

Comment:
Duration of source systems is hard to read.  Is "age" or "size" better or at leas more more tangible?